micromamba/src/umamba.cpp

```diff
@@ -79,19 +79,6 @@ set_umamba_command(CLI::App* com)
     CLI::App* env_subcom = com->add_subcommand("env", "List environments");
     set_env_command(env_subcom);
 
-    std::stringstream footer;  // just for the help text
-
-    footer
-        << "In order to use activate and deactivate you need to initialize your shell.\n"
-        << "Either call shell init ... or execute the shell hook directly:\n\n"
-        << "    $ eval \"$(./micromamba shell hook -s bash)\"\n\n"
-        << "and then run activate and deactivate like so:\n\n"
-        << "    $ micromamba activate  " << termcolor::white
-        << "# notice the missing dot in front of the command\n\n"
-        << "To permanently initialize your shell, use shell init like so:\n\n"
-        << "    $ ./micromamba shell init -s bash -p /home/$USER/micromamba \n\n"
-        << "and restart your shell. Supported shells are bash, zsh, xonsh, cmd.exe, powershell, and fish."
-        << termcolor::reset;
-
-    com->footer(footer.str());
+    CLI::App* activate_subcom = com->add_subcommand("activate", "Activate an environment");
+    set_activate_command(activate_subcom);
```

Comment:
I'm still not 100% sure it will work fine on all platforms/shells (e.g. Windows/cmd.exe) if the executable is in the current working directory or found in the path

Reply:
this probably also needs an update of the shell completer subcommand https://github.com/mamba-org/mamba/blob/bab7ae50d75f067c493b8d0cd065ea29bd3f9e8c/micromamba/src/completer.cpp#L87

Reply:
> I'm still not 100% sure it will work fine on all platforms/shells...

I'm just trying to modify the error messages as per #1323... I don't understand yet how this could lead to breakage. I just want `./micromamba activate` to print a more sensible error.

> this probably also needs an update of the shell completer subcommand

Sounds good! Would you be able to help with that?

Reply:
> I don't understand yet how this could lead to breakage
I'm just not sure that calinng `micromamba activate` on `Windows/cmd.exe` always prioritize to the shell function instead of the executable. For other subcommands it's not a problem because the args are just forwarded to the executable, but for activation it could just make the shell function not reachable (masked) by the executable

> Sounds good! Would you be able to help with that?
yes I'll take a look